index.js

```diff
@@ -173,13 +274,13 @@ module.exports.toASCII = (domainName, useSTD3, processingOption, verifyDnsLength
   });
 
   if (verifyDnsLength) {
-    const total = labels.slice(0, labels.length - 1).join(".").length;
-    if (total.length > 253 || total.length === 0) {
```

Comment:
`total` is already a number -- the length of the domain name excluding the root label and its dot.